src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -15,18 +16,39 @@ import { ConversationSpace } from "components/templates/ConversationSpace";
 import { Embeddable } from "components/templates/Embeddable";
 import { FireBarrel } from "components/templates/FireBarrel";
 import { Jazzbar } from "components/templates/Jazzbar";
-import { PartyMap } from "components/templates/PartyMap";
 import { PlayaRouter } from "components/templates/Playa/Router";
 import { PosterHall } from "components/templates/PosterHall";
 import { PosterPage } from "components/templates/PosterPage";
 import { ScreeningRoom } from "components/templates/ScreeningRoom";
 import { ReactionPage } from "components/templates/ReactionPage";
 
-import { ChatSidebar } from "components/organisms/ChatSidebar";
-import { UserProfileModal } from "components/organisms/UserProfileModal";
-import { WithNavigationBar } from "components/organisms/WithNavigationBar";
-
-import { AnnouncementMessage } from "components/molecules/AnnouncementMessage";
+const PartyMap = React.lazy(() =>
+  import("components/templates/PartyMap").then((m) => ({ default: m.PartyMap }))
+);
+
+const ChatSidebar = React.lazy(() =>
+  import("components/organisms/ChatSidebar").then((m) => ({
+    default: m.ChatSidebar,
+  }))
+);
+
+const UserProfileModal = React.lazy(() =>
+  import("components/organisms/UserProfileModal").then((m) => ({
+    default: m.UserProfileModal,
+  }))
+);
+
+const WithNavigationBar = React.lazy(() =>
+  import("components/organisms/WithNavigationBar").then((m) => ({
+    default: m.WithNavigationBar,
+  }))
+);
+
+const AnnouncementMessage = React.lazy(() =>
+  import("components/molecules/AnnouncementMessage").then((m) => ({
+    default: m.AnnouncementMessage,
+  }))
+);
```

Comment:
Why were these particular components split with lazy loading? I'm not sure that they all make sense to.

Further, there will be conflicts with a bunch of these and the work I did on bundle splitting in:

- https://github.com/sparkletown/sparkle/pull/1737

So I would potentially suggest reverting these changes.

Reply:
Since aforementioned PR is merged in staging, b7ec26dc13ee3c39f4d061fe3565ff7cd02b0f3e should bring those component in line with it.